src/index.js

```diff
@@ -108,6 +185,9 @@ export default function transformCssModules({ types: t }) {
                     }
                 });
 
+                // wrap or define processCss function that collect generated css
+                currentConfig.processCss = pushStylesCreator(currentConfig.processCss);
```

Comment:
Indeed.

I updated `pushStylesCreator` to use `processCss` return value if it is a string.